Comparison Engine/Comparison Engine/Base Classes/BarManager.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Comparison_Engine.Base_Classes
+{
+    public sealed class BarManager
+    {
+        private static readonly Lazy<BarManager> lazy = new Lazy<BarManager>(() => new BarManager());
+
+        public static BarManager Instance { get { return lazy.Value; } }
+
+        private BarManager()
+        {
+
+        }
+
+        //A list of bars
+        public List<Bar> barList = new List<Bar>();
+
+        //Adds a new bar
+        public void AddBar(string barName, string location)
+        {
+            int barID = barList.Count - 1;
+            barList.Add(new Bar(barID, barName, location));
+        }
```

Comment:
We might need to have a lot more information about a bar than just the name and the location, but it's fine to portray a model. Not sure if "address" would be more appropriate instead of "location", but I guess it could give as much clarity as it could make a mess. Saying this based off the fact that I named all address strings in MapController, well, "address" :D So that's just a few thoughts for the future.

Reply:
Yes, once we get more information, will probably need to expand this to accommodate that.
Regarding the variable name, are we sticking to using addresses to the end? If so, then yes, it would make sense to change the name to "address"